### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sumukhaithal6**
-Exporting pytorch model to ONNX format
-Converting ONNX model to Tensorflow and saving it as .pb file
-Learning rate warmup

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -46,56 +52,107 @@ def _build_criteria_and_optim(self):
             self.model.parameters(), **self.get("optim/kwargs")
         )
 
+    def _build_scheduler(self):
+        self.scheduler_steplr = StepLR(self.optim, step_size=10, gamma=0.1)
+        self.scheduler_warmup = GradualWarmupScheduler(self.optim, multiplier=1, total_epoch=5, after_scheduler=self.scheduler_steplr)
```
Comment (nasimrahaman) [inline train.py:57]: Suggestion: rename `self.scheduler_warmup` to `self.scheduler`, and `self.scheduler_steplr` to `self._base_scheduler`.

File: train.py [hunk 2]
```diff
@@ -46,56 +52,107 @@ def _build_criteria_and_optim(self):
...
     @register_default_dispatch
     def train(self):
+        self.initialize_wandb()
         for epoch in self.progress(
             range(self.get("training/num_epochs", ensure_exists=True)), tag="epochs"
         ):
             self.train_epoch()
             validation_stats = self.validate_epoch()
             self.log_progress("epochs", **validation_stats)
+            self.scheduler_warmup.step(epoch)
```
Comment (nasimrahaman) [inline train.py:72]: This should be `self.scheduler.step(epoch)` after renaming.

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,7 @@
 torch
 unittest
 numpy
-addict
\ No newline at end of file
+addict
+tensorflow==1.15
```
Comment (nasimrahaman) [inline requirements.txt:5]: Question: will `tensorflow-cpu` be enough?

File: onnx_export.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import torch
```
Comment (nasimrahaman) [inline onnx_export.py:1]: Can we please make a directory called `scripts` and have this file in that directory?

File: tf_conversion.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import onnx
```
Comment (nasimrahaman) [inline tf_conversion.py:1]: Can we have this in a separate `scripts` directory?

File: requirements.txt (hunk 3 above)
Comment (sumukhaithal6) [reply requirements.txt:5]: Yes

File: onnx_export.py (hunk 4 above)
Comment (sumukhaithal6) [reply onnx_export.py]: Done
